indico/types/model.py

```diff
@@ -29,3 +32,34 @@ class Model(BaseType):
     id: int
     status: str
     training_progress: TrainingProgress
+
+
+class ModelOptions(BaseType):
```

Comment:
Should this be `ModelTrainingOptions` ?

Reply:
i thought the same too but saw that it had an attribute model_training_options. if this is supposed to be model training options maybe we can update the typing in update_model_group_settings to make creating the model options a bit easier on the user side